src/Core/src/Platform/ViewExtensions.cs

```diff
@@ -159,5 +185,36 @@ internal static bool IsLoadedOnPlatform(this IElement element)
 #endif
 
 		}
+
+
+#if PLATFORM
+		internal static T? FindDescendantView<T>(this PlatformView view, Func<PlatformView, bool> predicate) where T : PlatformView
+		{
+			var queue = new Queue<PlatformView>();
+			queue.Enqueue(view);
+
+			while (queue.Count > 0)
+			{
+				var descendantView = queue.Dequeue();
+
+				if (descendantView is T result && predicate.Invoke(result))
+					return result;
+
+				int i = 0;
+				PlatformView? child;
+				while ((child = descendantView?.GetChildAt<PlatformView>(i)) is not null)
+				{
+#if TIZEN
+					if (child is null)
```

Comment:
Can this happen with the while statement checking for null? Also, currently won't Tizen just throw an exception?

Reply:
I think there's something off with the null ref analyzer and Tizen. I had to add this check for Tizen to compile.  I added a comment to the code